runtime/fastly/host-api/fastly.h

```diff
@@ -7,10 +7,39 @@ extern "C" {
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
+namespace fastly {
 
-#include "./component/fastly_world.h"
+typedef struct fastly_world_string {
+  uint8_t *ptr;
+  size_t len;
+} fastly_world_string;
```

Comment:
It would be nice to see these removed entirely with less ceremony / better internal types, but we can always do that as a follow-up.